waves/package.json

```diff
@@ -35,7 +35,7 @@
     "web-vitals": "^0.2.4"
   },
   "scripts": {
-    "start": "react-app-rewired start",
+    "start": "PORT=3004 react-app-rewired start",
```

Comment:
Why can this not be done in the script instead?

Reply:
Also, would `REACT_APP_PORT` work here to be less ambiguous?

Reply:
> Why can this not be done in the script instead?

I tried different combinations, but only this worked. I couldn't figure out how to forward this env variable, but it was getting late.

Reply:
> Also, would `REACT_APP_PORT` work here to be less ambiguous?

The environment variable that is expected here is `PORT`, `REACT_APP_PORT` doesn't do anything :(